src/components/Account/Account.tsx

```diff
@@ -288,6 +289,28 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
                 }),
               ]
             : []),
+          ...(canDeleteAccount
```

Comment:
I would prefer doing it this way:  `canDeleteAccount && formSection({...`
And then we could filter the array using `filter(Boolean)` method.

Reply:
I meant to add `filter(Boolean)` to the whole array of sections:

```
[
formSection({...,
canDeleteAccount && formSection({...
].filter(Boolean)
```

Reply:
I can't seem to make this work, typescript is complaining about false still being a potential value. Could you try and suggest it as a commit exactly?